### DISCUSSION TITLE
Login issue in SnipeIT

### DISCUSSION THREAD

**Discussion by jainjine** [snipe/snipe-it#10227] [General]
Hi,

I have install snipeIT on my server and I have used first 2 weeks and after it was ideal mode  (not used actively). Later 1 month I have login with my credential and cannot login I have checked mysql tables and I could not see the which I have created and also cannot login with admin credentials.

what is this happened? it is DB hack? or what else? 

could you please tell me how to  recover my admin login for the same DB?

Thanks
Jineesh

**Comment by snipe** [comment]
You can see how it would be impossible for me to determine if your database was hacked based on the information you've provided - but odds are, it wasn't. You can create a rescue admin account by following the steps here: https://snipe-it.readme.io/docs/create-an-admin

**Reply by jainjine** [reply]
Hi Thanks.! I have restored this.
